var/spack/repos/builtin/packages/nrm/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Nrm(PythonPackage):
+    """Node Resource Manager"""
+
+    homepage = "https://xgitlab.cels.anl.gov/argo/nrm"
+    url = "https://www.mcs.anl.gov/research/projects/argo/downloads/nrm-0.1.0.tar.gz"
+    version('0.1.0', '99a8c8d312fcf3880608cdc8274c285f')
+
+    depends_on('py-setuptools', type=('build'))
+
+    depends_on('py-six', type=('build', 'run'))
+    depends_on('py-zmq', type=('build', 'run'))
+    depends_on('py-pyyaml', type=('build', 'run'))
+    depends_on('py-tornado', type=('build', 'run'))
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-argparse@1.2.1:', type=('build', 'run'))
+    depends_on('py-jsonschema', type=('build', 'run'))
+    depends_on('py-warlock', type=('build', 'run'))
+    depends_on('py-scipy', type=('build', 'run'))
```

Comment:
None of these dependencies are listed in the `setup.py`.

Reply:
We like to use `requirements.txt`. Is this prohibitive for the Spack package to work? I'm happy adding them.

Reply:
Not prohibitive, just unusual since other package managers like `pip` don't use `requirements.txt`, they rely on dependencies explicitly mentioned in `setup.py`. I'll trust that you know what you're doing.